lib/salus/report.rb

```diff
@@ -334,7 +373,12 @@ def report_body(config)
         body = report_body_hash(config, JSON.parse(to_sarif(config['sarif_options'] || {})))
       end
       body = report_body_hash(config, JSON.parse(to_sarif_diff)) if config['format'] == 'sarif_diff'
-      return JSON.pretty_generate(body) if %w[json sarif sarif_diff].include?(config['format'])
+      if config['format'] == 'cyclonedx-json'
+        body = report_body_hash(config, JSON.parse(to_cyclonedx(config['cyclonedx_options'] || {})))
```

Comment:
Want to add in config['cyclonedx_options'] support in a future PR for CycloneDX Spec versions.